service/translations-example.gradle

```diff
@@ -0,0 +1,42 @@
+android {
```

Comment:
There is also a translations.gradle-example file, which can be removed/renamed. We don't need both of them.

Reply:
I've removed the unnecessary file.